src/components/Header/Header.js

```diff
@@ -30,13 +30,16 @@ const Header = React.memo(function Header({ compactMode, toggleMenuPanel }) {
       <div
         css={`
           display: flex;
-          flex-grow: 1;
-          justify-content: center;
+          align-self: stretch;
+
+          & > * {
+            align-self: stretch;
+          }
```

Comment:
Is this needed?

Reply:
Not at all, forgot to remove it when updating the inner components. The `align-self` property on the parent component can be replaced with `height: 100%`